pkg/runtime/runtime.go

```diff
@@ -1953,11 +1953,14 @@ func (a *DaprRuntime) BulkPublish(req *pubsub.BulkPublishRequest) (pubsub.BulkPu
 	if allowed := a.isPubSubOperationAllowed(req.PubsubName, req.Topic, ps.scopedPublishings); !allowed {
 		return pubsub.BulkPublishResponse{}, runtimePubsub.NotAllowedError{Topic: req.Topic, ID: a.runtimeConfig.ID}
 	}
+	policyDef := a.resiliency.ComponentOutboundPolicy(req.PubsubName, resiliency.Pubsub)
 	if bulkPublisher, ok := ps.component.(pubsub.BulkPublisher); ok {
-		return bulkPublisher.BulkPublish(context.TODO(), req)
+		return runtimePubsub.ApplyBulkPublishResiliency(a.ctx, req, policyDef, bulkPublisher)
 	}
 	log.Debugf("pubsub %s does not implement the bulkPublish API, defaulting to normal publish", req.PubsubName)
```

Comment:
```suggestion
	log.Debugf("PubSub %s does not implement the BulkPublish API; falling back to publishing messages individually in sequence", req.PubsubName)
```

Reply:
partially used this. all other debug messages start with lowercase char. and we use `pubsub` a lot in other places also.